src/pybind/mgr/dashboard/frontend/src/app/shared/classes/crush.node.selection.class.ts

```diff
@@ -19,6 +19,7 @@ export class CrushNodeSelectionClass {
   failureDomainKeys: string[] = [];
   devices: string[] = [];
   deviceCount = 0;
+  autoDeviceUpdate: boolean = true;
```

Comment:
Maybe a coment here to explain why is this, would be nice for future us/maintainers